src/Stryker.Core/Stryker.Core/Mutators/LinqMutator.cs

```diff
@@ -33,23 +34,25 @@ static LinqMutator()
                 { LinqExpression.TakeWhile, LinqExpression.SkipWhile },
                 { LinqExpression.Min, LinqExpression.Max },
                 { LinqExpression.Max, LinqExpression.Min },
-                { LinqExpression.Sum, LinqExpression.Count },
+                { LinqExpression.Sum, LinqExpression.Max },
```

Comment:
The documentation isn't updated to the new mutations

Reply:
Thanks,
updated doc with note explaining the change.